activemodel/lib/active_model/dirty.rb

```diff
@@ -189,7 +189,7 @@ def attribute_was(attr) # :nodoc:
     end
 
     # Handles <tt>*_previously_changed?</tt> for +method_missing+.
-    def attribute_previously_changed?(attr, options = {}) #:nodoc:
+    def attribute_previously_changed?(attr) #:nodoc:
```

Comment:
@vipulnsward did it in his alternative PR for some reason and recommended it to be used in mine. Sorry, I didn't think it through myself. @vipulnsward can you explain it?